test/pycheck/explain_test.py

```diff
@@ -84,6 +86,38 @@ def test_explain(cur: Cursor):
     assert "Execution Time" in (result[0]).keys()
 
 
+def test_explain_execute_statement(cur: Cursor):
+    """
+    EXPLAIN EXECUTE is known to be broken, because we don't have the right
+    hooks in Postgres 17 and below. We should fix this for PG18.
+
+    For new we at least run it to make sure that it doesn't crash (which it did
+    in the past).
+    """
+    cur.sql("CREATE TABLE test_table (id int, name text)")
+    cur.sql("EXPLAIN (ANALYZE, FORMAT JSON) SELECT * FROM test_table t")
+    cur.sql("PREPARE test_query AS SELECT * FROM test_table t")
+    result = cur.sql("EXPLAIN EXECUTE test_query")
+    plan = "\n".join(result)
+    print(plan)
+    result2 = cur.sql("EXPLAIN EXECUTE test_query")
+    plan2 = "\n".join(result2)
+    print(plan2)
```

Comment:
Can we make some assertions to the plans rather than printing them out? (or only print them when a debug mode is turned on?)